spec/models/category_spec.rb

```diff
@@ -1,5 +1,70 @@
-require 'spec_helper'
+require 'rails_helper'
 
-describe Category do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+RSpec.describe Category, :type => :model do
+
+  it { should have_many(:videos) }
```

Comment:
What do you mean?
I see the tutorial in shoulda-matcher. It use should syntax.
'''
describe Person do
  it { should validate_presence_of(:name) }
end
'''